### PR TITLE
Add debugging context with eventId

### PR DESCRIPTION
**Pull request by jekelly-adobe**
https://jira.corp.magento.com/browse/DINT-146

Goes with https://github.com/adobe/magento-storefront-event-collector/pull/102

### CODE REVIEW

File: src/types/schemas/debug.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type Debug = {
+    eventId?: string;
```
Comment (aniham) [inline src/types/schemas/debug.ts:2]: no need to change - but might be good to not have this as optional

**Review by aniham — APPROVED** [review]
@jekelly-adobe awesome. i'm so happy about this. i think this is going to grow into other things too (request id, other debug info) that can be very powerful

File: src/types/schemas/debug.ts (hunk 1 above)
Comment (jekelly-adobe) [reply src/types/schemas/debug.ts:2]: An optional value lets us *not* specify an eventId in order to indicate that event IDs should not be overwritten with custom values. Why might we want it to be required?

Comment (aniham) [reply src/types/schemas/debug.ts:2]: got it - i was thinking about why we'd set the debug context as empty - but it makes sense thinking about leaving it empty actually having a meaning!!

**Pull request merged**